src/Analyzers/CSharp/CodeFixes/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionCodeFixProvider.cs

```diff
@@ -94,10 +99,10 @@ protected override async Task FixAllAsync(Document document, ImmutableArray<Diag
             }
         }
 
-        private sealed class MyCodeAction : CodeAction.DocumentChangeAction
+        private sealed class MyCodeAction : CustomCodeActions.DocumentChangeAction
```

Comment:
Why this change?

Reply:
`DocumentChangeAction` is an internal nested type define within `CodeAction` a public type. It is not possible to just split the nested type into a separate file and move it to shared layer due to the outer type being a public type. A better approach is to clone this into a separate internal `CustomCodeActions` type which is added to the shared WorkspaceExtensions project so it can be referenced from both Features and CodeStyle.Fixes projects. We should eventually be able to get rid of CodeAction.DocumentChangeAction once our IDE code fixers have moved to the new type.